src/js/components/sharedComponents/SVGLine.jsx

```diff
@@ -209,7 +209,7 @@ export default class SVGLine extends Component {
                 tabIndex="0"
                 className={classname}
                 x={this.state[`${data}TextX`]}
-                y={this.state[`${data}TextY`]}
+                y={this.state[`${data}TextY`] || this.props.textY}
```

Comment:
Why aren't these in initial state?

Also makes it really hard to know whats going on when we're dynamically accessing the state properties like this. Couldn't we just hard code it so we do `this.state.dataTextY/X`?

Seems strange also that we're using props some of the time and state some of the time?

Reply:
Yeah, this allows for developers to use multiple words and multiple text elements.  We use the props for the initial render.

Reply:
This is an anti-pattern. We should be using either props or state but not both.